pandas/tests/groupby/test_nunique.py

```diff
@@ -172,7 +172,7 @@ def test_nunique_preserves_column_level_names():
     # GH 23222
     test = DataFrame([1, 2, 2], columns=pd.Index(["A"], name="level_0"))
     result = test.groupby([0, 0, 0]).nunique()
-    expected = DataFrame([2], columns=test.columns)
+    expected = DataFrame([2], index=np.array([0], dtype=np.int_), columns=test.columns)
```

Comment:
is the np.int_ here needed?  ive been assuming that the `index=np.array([0])` ive been seeing has been there to ensure the indexes get np.int_ dtype.

Reply:
No, not needed anyore. This is from before #49815 was merged. I'll revert it.